.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     environment:
       S2I_VERSION: "1.1.7-226afa1"
       DOCKER_COMPOSE_VERSION: "1.16.1"
-      OPENRESTY_VERSION: "1.11.2.5-1"
+      OPENRESTY_VERSION: "rover"
```

Comment:
A few lines below there's: `quay.io/3scale/s2i-openresty-centos7:1.11.2.5-2`. (Notice that the openresty version doesn't match :scream: )
Should it be adpated to use `rover` as well?

Reply:
Also, with this change, we'll no longer have documented the openresty version we are using.
Maybe we should use something like "rover-1.11.2.5-1" ?

Reply:
Yeah. I don't intend to merge it pointing to rover. After this is ready I'll merge the rover branch and do a new release of the s2i image with a correct version.